app/main/routes.py

```diff
@@ -387,26 +389,32 @@ def search_results_summary():
         or request.args.get("query", "").lower()
     )
     page = int(request.args.get("page", 1))
-
     filters = {"query": query}
+    search_results = None
+    num_records_found = 0
 
-    query = build_fuzzy_search_summary_query(query)
-    search_results = query.paginate(page=page, per_page=per_page)
+    if query.strip():
+        fuzzy_search_summary_query = build_fuzzy_search_summary_query(query)
+        search_results = fuzzy_search_summary_query.paginate(
+            page=page, per_page=per_page
+        )
 
-    total_records = db.session.query(
-        func.sum(query.subquery().c.records_held)
-    ).scalar()
-    if total_records:
-        num_records_found = total_records
-    else:
-        num_records_found = 0
+        total_records = db.session.query(
+            func.sum(fuzzy_search_summary_query.subquery().c.records_held)
```

Comment:
Is the following sufficient

```
total_records = db.session.query(
    func.sum(fuzzy_search_summary_query.records_held)
).scalar()
```

Reply:
the scalar() query return 'None' if no data returned, and will cause to overwrite num_records_found value of zero , which will show 'None' in frontend

Reply:
not sure if I follow, will ping you